queries.sql

```diff
@@ -134,3 +134,9 @@ WHERE vets.name = 'Maisy Smith'
 GROUP BY vets.name, animals.species_id, species.name
 ORDER BY animals.species_id DESC
 LIMIT 1;
+
+
+/**query analyze script**/
+EXPLAIN ANALYZE SELECT * FROM visits WHERE animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits WHERE vet_id = 2;
```

Comment:
### Optional
- [ ] It would be better if the time of execution of our `after query = 453.481` should be significantly lower than the `before query = 348.134` for our second query where `vet_id= 2`. In this case, I should suggest you use the Indexes on our foreign keys but in our case you have created the Indexes yet. Correct me if I'm wrong, below is the screenshot of the requirements
![Screenshot from 2022-11-01 16-17-25](https://user-images.githubusercontent.com/46388113/199283327-7ead04a8-3e08-4342-98dd-9630478e9afa.png)

Reply:
@addod19 Thanks for your detailed feedback on the optional suggestion you have given on that. I will definitely make these changes.